data/schemas/node-type-genspec.schema

```diff
@@ -417,6 +423,12 @@
           "minItems": 1,
           "uniqueItems": true,
           "items": { "$ref": "#/definitions/node" }
+        },
+        "aliases": {
+          "type": "array",
```

Comment:
Why not the `"$ref": "#/definitions/node"`way, instead of duplicating these options?